tests/publiccloud/az_accelerated_net.pm

```diff
@@ -0,0 +1,98 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Network performance for Azure Accelerated NICs
+#
+# Maintainer: Jose Lausuch <jalausuch@suse.de>
+
+use base "publiccloud::basetest";
+use strict;
+use testapi;
+use utils;
+use Data::Dumper;
+
+sub enable_accelerated_net {
+    my ($self, $instance) = @_;
+    my $name = $instance->{instance_id};
+
+    record_info('INFO', "Enabling accelerated network on instance '$name' ...");
+    assert_script_run("az vm deallocate --resource-group $name --name $name",                                        timeout => 60 * 10);
+    assert_script_run("az network nic update --name $name-nic --resource-group $name --accelerated-networking true", timeout => 60 * 10);
+    assert_script_run("az vm start --resource-group $name --name $name",                                             timeout => 60 * 10);
+}
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    my $provider = $self->{provider} = $self->provider_factory();
+    $provider->init();
+
+    my $instance1 = $provider->create_instance(image => $self->get_image_id($provider), instance_type => 'Standard_DS2_v2');
```

Comment:
I think at least the instance_type of the SUT VM should be configurable via `PUBLIC_CLOUD_INSTANCE_TYPE`, otherwise we need to adopt always the test module if we change the type.
If you need to check for certain VM types, maybe there is a generic way to know if the VM support accelerated networkd. Otherwise I would just add a list of known VM types.

Reply:
ok, will create a new machine with PUBLIC_CLOUD_INSTANCE_TYPE=Standard_DS2_v2